src/post.builds

```diff
@@ -15,4 +15,20 @@
   <Target Name="Rebuild" DependsOnTargets="Clean;Build" />
   <Target Name="DumpTargets" />
 
+  <Target Name="CopyContractsToCompatFolder" AfterTargets="Build">
+    <ItemGroup>
+      <AllCompatAssemblies Include="$(BaseOutputPath)/**/Compat/*.dll" />
+      <AllCompatAssemblies>
+        <IsRefAssembly  Condition="$([System.String]::Copy('%(AllCompatAssemblies.FullPath)').Contains('/ref/')) OR
+                                   $([System.String]::Copy('%(AllCompatAssemblies.FullPath)').Contains('\ref\'))">true</IsRefAssembly>
+        <IsIntermediateAssembly Condition="$([System.String]::Copy('%(AllCompatAssemblies.FullPath)').Contains('/obj/')) OR
+                                   $([System.String]::Copy('%(AllCompatAssemblies.FullPath)').Contains('\obj\'))">true</IsIntermediateAssembly>
+      </AllCompatAssemblies>
+      <AllCompatAssemblies Remove="@(AllCompatAssemblies)" Condition="'%(AllCompatAssemblies.IsIntermediateAssembly)'=='true'" />
+    </ItemGroup>
+
+    <Copy SourceFiles="@(AllCompatAssemblies)" Condition="'%(AllCompatAssemblies.IsRefAssembly)'=='true'" DestinationFolder="$(BaseOutputPath)/Compat/ref" />
```

Comment:
I can't do that mainly for two reasons:
- If I use `/obj/**/Compat/*.dll` for the facades then it will also bring the ref assemblies since they are located at `/obj/ref/**/Compat/*.dll`, although I suppose there are ways to work around this.
- In order to have a full Api Closure, I'm going to copy the reference assemblies where the facade's type forwards are defined, and they are being dropped at the OutDir, as opposed to IntermediateOutputDir.